include/dlaf/eigensolver/band_to_tridiag/mc.h

```diff
@@ -1040,14 +1041,14 @@ TridiagResult<T, Device::CPU> BandToTridiag<Backend::MC, D, T>::call_L(
   // The offset is set to the first unused tag by compute_copy_tag.
   const comm::IndexT_MPI offset_v_tag = compute_copy_tag(nrtile, false);
 
-  auto compute_v_tag = [offset_v_tag](SizeType i) {
+  auto compute_v_tag = [offset_v_tag](SizeType i, bool is_bottom) {
```

Comment:
Maybe consider an enum to replace the `bool is_bottom`, but not a blocker.

Reply:
I don't like much the idea to add enums for this kind of cases (lambda in a specific implemetation). Moreover this is a workaround to make the HH copy part working as it is. Anyway it still need to be improved.

Reply:
Ok, fair enough, I'm not insisting. For a more permanent solution I think it would aid readability with an enum (since the choice of `bool bottom` as the function parameters seems arbitrary and might as well be `bool top`), even if it's an enum only for use for this algorithm.